### PR TITLE
Improve error visibility

### PR DESCRIPTION
**Pull request by SCdF**
An attempt to help diagnose medic/medic-webapp#2774:
- Errors are outputted in red
- At the end of a run, if there are any errors another error is outputted so people don't miss it

### CODE REVIEW

File: import [hunk 1]
```diff
@@ -332,16 +332,20 @@ var onDataHandler = function(obj) {
...
+  });
 
-process.on("exit", function() {
- console.info("");
- console.info(stats);
+process.on('exit', function() {
+  console.info('');
+  console.info(stats);
+
+  if (stats.responses && (!stats.responses['200'] || Object.keys(stats.responses).length > 1)) {
```
Comment (mandric) [inline import]: Only problem here is we get 200 and 201s from couchdb that are success.  So we might test for 201 also or have a for loop that does a `if (retval < 200 || retval > 299) { ... }` or something.

**Comment by mandric** [comment]
I think there's one [minor issue](https://github.com/medic/medic-import/pull/3/commits/d02e5b8e5610325056b3aae059eec310ab3559f9#r83299269) with the error reporting code but otherwise looks good.  Feel free to merge, or pass back to me again if you prefer.

File: import (hunk 1 above)
Comment (SCdF) [reply import]: Nice catch. I'll refactor that to be a bit more robust.

**Comment by SCdF** [comment]
@mandric updated it to error is there are any non 2xx codes in the responses map. Take a look and merge if you're cool widdit

**Comment by mandric** [comment]
Awesome, thanks!

**Pull request merged**